src/main/java/de/bonndan/nivio/output/map/svg/SVGItem.java

```diff
@@ -81,11 +81,11 @@ public DomContent render() {
             circle.attr("stroke-dasharray", 5);
             circle.attr("opacity", 0.7);
         }
-        ContainerTag inner = SvgTagCreator.g(circle, content, children)
-                .attr("class", "hexagon");
+        ContainerTag inner = SvgTagCreator.g(circle, content, children);
 
         return SvgTagCreator.g(inner, icon)
-                .attr("class", "hexagon-group")
```

Comment:
Was this what made the hexagons before? Are we not having the hexagons anymore?

Reply:
Starting in line 92 in this class there is a TODO and a commented private method. Is this dead code? Do we need it, or can we remove it?

Reply:
The item is neither a "hexagon" anymore nor do we need the class here anymore.

The dead code has been copied to https://github.com/dedica-team/nivio/issues/303